### ISSUE TITLE
Loosing Connection When Docker Used

### ISSUE THREAD

**Issue by jyksnw** [slackhq/nebula#162]
Nebula Version: 1.0.0
Docker version: 19.03.3, build a872fc2f86
Host: Pop!_OS 19.04

I keep running into an issue where I loose my connection to one of my nodes when that node is running or has recently run a number of docker containers. When this happens I can't ping that node or access any of the ports I have open for the configuration. I am able to ping the host and access the ports when on the nodes local network.

Looking at the logs I can see that nebula is sending and receiving handshake messages. I am not sure what other information is needed but I will gladly dig into the logs and config to help resolve this issue.

As a work around I am now deploying my docker stuff into a VM that is hosted on that node.

**Comment by zeisss** [comment]
Just to be sure: Your docker IP range is not conflicting with your nebula IPs, right? IIRC docker plays around with iptable rules actively, so your problem might be related to that.

**Comment by jyksnw** [comment]
That could entirely be possible. I will re-run my tests tomorrow on the host machine to and check to see if that is what is happening.

**Comment by jyksnw** [comment]
It looks like that was it. I will close this issue since I don't believe there is anything that can be done on the nebula side to prevent this from happening.

**Comment by junky89** [comment]
I'm having this issue, and I tried with ranges 192.168.0.0/16, 10.10.0.0/16, 10.10.100.0/24 without success. Not sure which range to try. I can reproduce this with only 3 docker nodes. One being the lighthouse and let's say A and B. Lighthouse is always responding.  A an B depends on which one is connected last (last one can ping). If for example B is responding and A not, if I restart A, then the responding one is A. 

I tried ping A and B from the lighthouse node and also connecting the host machine to nebula. Same result. 

Any ideas?
